UnitsNet/GeneratedCode/Quantities/AmountOfSubstance.NetFramework.g.cs

```diff
@@ -859,6 +859,60 @@ public string ToString([CanBeNull] IFormatProvider provider, [NotNull] string fo
             return string.Format(provider, format, formatArgs);
         }
 
+        /// <summary>
+        /// Gets the string representation of this instance in the specified format string using <see cref="GlobalConfiguration.DefaultCulture" />.
+        /// </summary>
+        /// <param name="format">The format string.</param>
+        /// <returns>The string representation.</returns>
+        public string ToString(string format)
+        {
+            return ToString(format, GlobalConfiguration.DefaultCulture);
+        }
+
+        /// <summary>
+        /// Gets the string representation of this instance in the specified format string using the specified format provider, or <see cref="GlobalConfiguration.DefaultCulture" /> if null.
+        /// </summary>
+        /// <param name="format">The format string.</param>
+        /// <param name="formatProvider">Format to use for localization and number formatting. Defaults to <see cref="GlobalConfiguration.DefaultCulture" /> if null.</param>
+        /// <returns>The string representation.</returns>
+        public string ToString(string format, IFormatProvider formatProvider)
+        {
+            formatProvider = formatProvider ?? GlobalConfiguration.DefaultCulture;
+
+            int digits = 0;
+            string formatString = format;
+
+            if(string.IsNullOrWhiteSpace(formatString))
```

Comment:
NullOrEmpty. If format does not match any specifiers then it should return the format string as-is, if I'm reading the docs right. See earlier comment.

Reply:
I think FormatException is more widely used. See my earlier comment.